lab_3_keywords_textrank/main.py

```diff
@@ -278,7 +321,10 @@ def calculate_inout_score(self, vertex: int) -> int:
                 number of incidental vertices
         If vertex is not present in the graph, -1 is returned
         """
-        pass
+        if vertex not in self._matrix[0]:
+            return -1
+        idx = self._matrix[0].index(vertex)
+        return self._matrix[idx + 1][1:].count(1)
```

Comment:
i honestly didn't think of sum at all, aren't both of them approximately the same in terms of speed as both iterate the list? fix me if i'm wrong. sum must probably be used in case we have various weights and have to count like the total weight of a token, summing not just 1's but different integers

Reply:
I was thinking that using count except for iterating over the list we also perform a comparison to decide whether to count this element or not 
sum indeed summarizes all the elements in the array regardless of its value 
thus, given the fact that we are guaranteed to only have 0s and 1s in the matrix, would sum be more appropriate?

Reply:
k! fixed